contracts/Comptroller.sol

```diff
@@ -1135,9 +1049,23 @@ contract Comptroller is ComptrollerV4Storage, ComptrollerInterface, ComptrollerE
         return state;
     }
 
-    function _become(Unitroller unitroller) public {
+    function _become(Unitroller unitroller, uint vestingPeriod_) public {
         require(msg.sender == unitroller.admin(), "only unitroller admin can change brains");
         require(unitroller._acceptImplementation() == 0, "change not authorized");
+
+        Comptroller(address(unitroller))._becomeG6(vestingPeriod_);
```

Comment:
Is this meant to be G6 or G5? Does that deserve to be hard-coded in the comptroller.sol, or in the generation file?

Reply:
This looks right - G5 was borrow caps and this is the pattern we've used for initializing the impl version

Reply:
I thought Borrow Caps was G4...?

Reply:
Ah, right. So we keep all the 'G' contracts in the repo for the upgrade path, and G4 is the latest 'G' contract in the borrow caps branch, but the current impl is always called 'Comptroller.sol', so when we bump the next version it gets G5, which would make the current version G6 (even though it won't get renamed that until G7 comes out).